code/game/machinery/cloning.dm

```diff
@@ -166,8 +166,8 @@
 	if(efficiency > 2)
 		for(var/A in bad_se_blocks)
 			setblock(H.dna.struc_enzymes, A, construct_block(0,2))
-	if(efficiency > 5 && prob(20))
-		randmutg(H)
+	if(efficiency > 3 && prob(20))
+		randmutvg(H)
 	if(efficiency < 3 && prob(50))
 		var/mob/M = randmutb(H)
 		if(ismob(M))

```

Comment:
I do not believe this statement will ever be true - `randmutb(H)` does not return a mob, but a 1 or a 0. Try testing it with just `randmutb(H)`.